src/shared/components/ui/tabs/TabsTrigger.tsx

```diff
@@ -0,0 +1,72 @@
+import { ReactNode } from 'react';
+
+import { cn } from '@/shared/libs/cn';
+
+import { useTabs } from './useTabs';
+
+/**
+ * TabsTrigger 컴포넌트의 props 인터페이스
+ */
+interface TabsTriggerProps {
+  /** 탭을 식별하는 고유한 값 (TabsContent의 value와 매칭됨) */
+  value: string;
+  /** 탭 버튼에 표시할 내용 */
+  children: ReactNode;
+  /** 추가 CSS 클래스명 */
+  className?: string;
+}
+
+/**
+ * TabsTrigger 컴포넌트
+ *
+ * 개별 탭 버튼을 나타냅니다. 클릭하면 해당 value에 해당하는 TabsContent가 표시됩니다.
+ * 접근성을 위한 ARIA 속성과 키보드 네비게이션을 지원합니다.
+ *
+ * **접근성 기능:**
+ * - `role="tab"`: 스크린 리더에 탭 역할임을 알림
+ * - `aria-selected`: 현재 선택 상태를 나타냄
+ * - `tabIndex`: 활성 탭만 Tab 키로 접근 가능, 비활성 탭은 화살표 키로만 접근
+ * - `data-value`: 키보드 네비게이션에서 탭 식별용
+ * - `data-state`: CSS 스타일링용 상태 표시
+ *
+ * @example
+ * ```tsx
+ * <TabsTrigger
+ *   value="tab1"
+ *   className="px-4 py-2 data-[state=active]:bg-blue-100"
+ * >
+ *   첫 번째 탭
+ * </TabsTrigger>
+ * ```
+ *
+ * @param props - TabsTrigger 컴포넌트의 props
+ * @returns JSX 엘리먼트
+ */
+export default function TabsTrigger({
+  value,
+  children,
+  className,
+}: TabsTriggerProps) {
+  const { value: selectedValue, onValueChange } = useTabs();
+  const isSelected = selectedValue === value;
+
+  return (
+    <button
+      role='tab'
+      aria-selected={isSelected}
+      tabIndex={isSelected ? 0 : -1}
+      data-value={value}
+      onClick={() => onValueChange?.(value)}
+      className={cn(
+        'border-b border-b-white px-10 py-5 text-black',
+        isSelected
+          ? 'text-brand-2 border-b-brand-2 border font-semibold'
+          : 'text-gray-500',
+        className,
+      )}
+      data-state={isSelected ? 'active' : 'inactive'}
```

Comment:
`data-state={isSelected ? 'active' : 'inactive'}`로써 CSS에서 선택자를 [data-state="active"]로 사용하여 상태에 따른 스타일링을 할 수 있어 좋은 패턴인 거 같습니다!

현재 코드에서 `isSelected `삼항 연산자와 `data-state`가 모두 사용되고 있는데, `cn` 함수와 `data-state`를 활용하여 스타일 로직을 일관되게 관리하면 더 깔끔해질 거 같습니다.

```suggestion
      className={cn(
    'border-b border-b-white px-10 py-5 text-gray-500',
    'data-[state=active]:text-brand-2 data-[state=active]:border-b-brand-2 data-[state=active]:font-semibold',
    className,
)}
```